pm_icecon/nt/compute_nt_ic.py

```diff
@@ -177,34 +177,93 @@ def apply_nt_spillover(
     *, conc: npt.NDArray, shoremap: npt.NDArray, minic: npt.NDArray
 ) -> npt.NDArray[np.int16]:
     """Apply the NASA Team land spillover routine."""
+    """
+      conc is 0 to > 100.0
+        (per this code, negative value means missing data)
+
+      shoremap is 0: ocean, 1: land, 2: coast,
+                   3: coast, 4: near_coast, 5: far_coast
+        in original, shoremap is big-endian int16, but nothing in the
+          computation requires it to be anything other than int-like
+
+      minic is 0 to 100.0
+        (in original binary file, minic is 0-1000 which is conc * 10)
+        but as included here, it is float64 with values 0 to 96.3 (=~100.0)
+
+      It appears that there is an assumption that land is always >= 15% conc
+      TODO: Perhaps this should be looking for ocean cells that are >= 15%?
+    """
     newice = conc.copy()
 
-    is_at_coast = shoremap == 5
+    is_at_coast = shoremap == 3
     is_near_coast = shoremap == 4
-    is_far_coastal = shoremap == 3
+    is_far_coastal = shoremap == 5
 
     mod_minic = minic.copy()
-    mod_minic[is_at_coast & (minic > 20)] = 20
+    mod_minic[is_at_coast & (minic > 60)] = 60
     mod_minic[is_near_coast & (minic > 40)] = 40
-    mod_minic[is_far_coastal & (minic > 60)] = 60
+    mod_minic[is_far_coastal & (minic > 20)] = 20
 
     # Count number of nearby low ice conc
     n_low = np.zeros_like(conc, dtype=np.uint8)
 
+    # TODO: The original NASA code allows low-conc values over land to
+    #       count toward the number of nearby low-conc values needed to
+    #       cause a grid cell to be considered spillover.  This seems like
+    #       an error.
```

Comment:
Lets capture this (and any other TODO that is worth considering in the future) in an issue on this repo.

Reply:
https://github.com/nsidc/pm_icecon/issues/63